docbook/reference/en/en-US/modules/RESTEasy_Spring_Integration.xml

```diff
@@ -1,221 +1,416 @@
+    <artifactId>resteasy-spring-boot-starter</artifactId>
+    <version>${latest_version_of_restesy_spring_boot}</version>
+</dependency>
+                ]]></programlisting>
+            </para>
+            </listitem>
+        </itemizedlist>
+
+        <para>RESTEasy currently supports Spring version &SPRING-VERSION;</para>
+
         <para>
-            RESTEasy comes with its own <code>ContextLoaderListener</code> that registers a RESTEasy specific
-            <code>BeanPostProcessor</code>
-            that processes &REST-API; annotations when a bean is created by a <code>BeanFactory</code>. And it will
-            automatically scan for <code>@Provider</code> and &REST-API; resource annotations on your
-            bean class and
-            register them as &REST-API; resources.
+            RESTEasy provides integrated support for Spring Framework.
+            It supplies a default <code>org.springframework.web.context.ContextLoaderListener</code>
+            and <code>org.springframework.beans.factory.config.BeanPostProcessor</code>
+            implementation that is used to run Spring applications.  Alternatively
+            RESTEasy provides support for registering a custom
+            <code>org.springframework.beans.factory.config.BeanFactoryPostProcessor</code>
+            thus enabling the user to create their own bean factories.
         </para>
-        <para>Here is the content that you should add into your <code>web.xml</code> file:
+
+        <para>
+            Spring MVC Framework support is provided in RESTEasy.
+            A default implementation of Spring's <code>org.springframework.web.servlet.DispatcherServlet</code>
+            is supplied.
         </para>
-        <programlisting><![CDATA[<listener>
-    <listener-class>org.jboss.resteasy.plugins.server.servlet.ResteasyBootstrap</listener-class>
-</listener>
 
-<listener>
-    <listener-class>org.jboss.resteasy.plugins.spring.SpringContextLoaderListener</listener-class>
-</listener>]]></programlisting>
         <para>
-            Please note that the <code>SpringContextLoaderListener</code> must be declared after <code>
-            ResteasyBootstrap
-        </code> as it
-            uses
-            <code>ServletContext</code>
-            attributes initialized by it.
+            RESTEasy furnishes an Undertow based embedded Spring
+            container in which Spring applications can run.  It supplies
+            class <code>org.jboss.resteasy.plugins.server.undertow.spring.UndertowJaxrsSpringServer</code>
+            which accepts a Spring context configuration file and preforms the appropriate wiring
+            of Spring to RESTEasy.
         </para>
+
         <para>
-            And you can configure the Springframework to scan for the &REST-API; resources and beans in a Spring
-            configuration file. The content of the file is shown as follow:
+            Two Spring Boot starters are provided by resteasy-spring-boot.
```

Comment:
maybe here should be `two types of starters` or just `Spring Boot Starters are provided by ...`